.github/workflows/ci.yml

```diff
@@ -44,7 +44,8 @@ jobs:
           "cockroach:latest-v24.2",
           "cockroach:latest-v24.3",
           "cockroach:latest-v25.1",
```

Comment:
we can remove testing against v25.1, as it as reached end-of-life: https://www.cockroachlabs.com/docs/releases/release-support-policy#unsupported-versions